api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -73,9 +73,9 @@ type VirtualMachineImageStatus struct {
 	// Deprecated
 	PowerState    string `json:"powerState,omitempty"`
 
-	// GuestOSCustomizable indicates whether the VirtualMachineImage's osType is supported for
-	// customisation on the cluster
-	GuestOSCustomizable *bool  `json:"guestOsCustomizable,omitempty"`
+	// SupportedGuestOS indicates whether the VirtualMachineImage's osType is supported
+	// by the hosts in the cluster
+	SupportedGuestOS *bool  `json:"supportedGuestOs,omitempty"`
```

Comment:
I think this should be `supportedGuestOS`

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#constants 

>  When an acronym or initialism each letter in the acronym should be uppercase, such as with ClientIP or TCPDelay